0076-Hold_Rewards_for_Denylist_Hotspots.md

```diff
@@ -0,0 +1,22 @@
+Helium Improvement Proposal (HIP) Title: Temporarily Holding Rewards for Denied List Hotspots
+
+Authors:
+- Rob#8633
+- H3ct0R
+- ChatGPT
+
+## Introduction:
+The Helium network is a decentralized network that relies on hotspots to create and maintain the network. However, in some cases, the rewards of some hotspots may be denied due to potential fraudulent activity. While it is important to prevent fraud, it may also result in the unfair denial of rewards to legitimate hotspots.
+
+## Proposal:
+To address this issue, this HIP proposes a temporary holding of rewards for hotspots that have been placed on the denied list. The rewards would be held until a review and appeal process is completed. If the hotspot is deemed to be legitimate, the rewards would be returned to the hotspot owner.
+
```

Comment:
Does this mean that the pool of rewards are therefore not going to active hotspots on the network? Just on the surface, this looks like it would negatively impact all hotspots that are not on the denylist by limiting the pool of mining rewards that go to active and honest hotspots.

Reply:
I would think maybe it gets held for a period of time, maybe 60 days or something until the denylist hotspot owner has had chance to appeal the decision. After this the held funds if no appeal is granted could then be either burnt or donated to charity, or just evenly shared amongst legitimate online hotspots during this time as some form of "we caught some gamers bonus".

Reply:
Before rewards can be held in a reserve like this, they have to come from somewhere and be accounted for. 

The current pool of all rewards is [distributed across the network in defined buckets](https://docs.helium.com/blockchain/mining#hnt-distributions-per-epoch) so any additional portions would need to come from one of those existing buckets. Do you have a particular bucket in mind?

Reply:
Make a denylist submission put a staked bond of 5 HNT.  If you are found legit, you get it back.  If you are still found to be a gamer, you forfeited the 5 HNT.